h/realtime.py

```diff
@@ -85,8 +86,6 @@ def publish_user(self, payload):
         self._publish("user", payload)
 
     def _publish(self, routing_key, payload):
-        retry_policy = {"max_retries": 5, "interval_start": 0.2, "interval_step": 0.3}
```

Comment:
The actual policy isn't the one here, it's might tighter.

I'm honestly a bit confused about how these apply in reality. The `transport_options` attached to the connection object seem to be the ones that take effect in a loss of Rabbit. So are these additional retries over that retrying or retrying something else entirely?

I've timed this and it seems to have the 0.6s from the connection policy you'd expect. So honestly I'm not quite sure when this applies.